### ISSUE TITLE
Tribler doesn't close after report submission (after a crash)

### ISSUE THREAD

**Issue by ManInTheGitHub** [Tribler/tribler#2640]
- [x] I have read CONTRIBUTING.rst

- [x] I have tried with the latest devel version and I still can reproduce the issue.

##### Tribler version/branch+revision: #####
7.0.0-alpha1
##### Operating system and version: #####
Debian testing 64-bit
##### Expected behavior: #####
Tribler crashed (see submitted report); I clicked report: I now expect it to close.
##### Actual behavior: #####
GUI disappears, Tribler keeps running; tray icon does not have a left or right mouse button menu that I can use to exit, command line kill on "/usr/bin/python2.7 run_tribler.py" still didn't kill it. It is now detached from my terminal, but still running. A kill on the detached process doesn't kill it. A kill -9 does.
##### Steps to reproduce the behavior: #####
I did nothing to trigger it; it was running in the background.
##### Relevant log file output: #####
Unknown what the relevant part is. Tribler kept running after all and the log is big and does not have timestamps (please add them...)

**Comment by devos50** [comment]
https://github.com/Tribler/tribler/pull/2649/commits/f5f3b601ac46c94c7052819cd86308cef699ee7a contributes towards the fix of this issue (but does not fix it).

**Comment by devos50** [comment]
I think I've found the root issue of this: the stop timer is started when an exception is thrown (https://github.com/Tribler/tribler/blob/devel/TriblerGUI/core_manager.py#L138), causing `QApplication.quit` to be invoked. However, the presentation of the crash dialog is blocking (https://github.com/Tribler/tribler/blob/devel/TriblerGUI/tribler_window.py#L74), causing some kind of weird state and probably explaining why Tribler keeps running. This is probably also related to the issue where the `_popen` attribute of the multiprocessing process is not present (I've fixed that by using the shutdown endpoint in the API).
